### PR TITLE
feat(PP-62):changed country code display to country flags

### PR DESCRIPTION
**Pull request by DmitrijSidorkin**

### CODE REVIEW

File: views/partials/account-navigation.ejs [hunk 1]
```diff
@@ -5,10 +5,15 @@
...
     />
-    <div class="profile-name">
-      <%-profileData?.displayName || profileData?.username %>
+    <div class="profile-name-div">
+      <div class="profile-name">
+        <%-profileData?.displayName || profileData?.username %>
+      </div>
+      <img
+        id="user-country"
+        src="../../icons/flags/<%-profileData.country%>.png"
```
Comment (elnino97) [inline views/partials/account-navigation.ejs:14]: What happens if user's country code does not match any file names in the flags folder?

File: views/main/test.ejs [hunk 2]
```diff
@@ -1 +1,4 @@
 <div>TEST</div>
+<div id="test"></div>
```
Comment (elnino97) [inline views/main/test.ejs:2]: What is this for?

File: views/partials/account-navigation.ejs [hunk 3]
```diff
@@ -10,6 +10,7 @@
         <%-profileData?.displayName || profileData?.username %>
       </div>
       <img
+        onerror='this.style.display="none"'
```
Comment (elnino97) [inline views/partials/account-navigation.ejs:13]: I would prefer to add a condition around the `img` element checking if `profileData.country` or havinga placeholder icon

File: views/main/test.ejs (hunk 2 above)
Comment (DmitrijSidorkin) [reply views/main/test.ejs:2]: Used this to check what countries were missing flag icons. Is kinda leftover code, but this entire page is going away once we reach deployment either way.

**Comment by DmitrijSidorkin** [comment]
Made a .js file that performs a check whether a country flag image exists, and if not removes the `image` element.

**Comment by elnino97** [comment]
> Made a .js file that performs a check whether a country flag image exists, and if not removes the `image` element.

I think there's no need, you can simply do this:
```
<% if(profileData.country{ %>
      <img
        id="user-country"
        src="../../icons/flags/<%-profileData.country%>.png"
        />
<% } %>
```
Or is there a reason why you wouldn't want to?

**Comment by DmitrijSidorkin** [comment]
Kinda forgot there for a moment how ejs works. This is what coding at 8am does to a man xd

**Review by elnino97 — APPROVED** [review]

**Pull request merged**